task1/src/test/java/com/fundacionjala/movies/RentalTest.java

```diff
@@ -0,0 +1,40 @@
+package com.fundacionjala.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RentalTest {
```

Comment:
Improve this test class
add new unit tests.

Example:
testCalculateCharge
testCalculateRenterPoints
testCalculateRenterPointsNewRelease